src/modules/imageresizer/ui/Properties/Settings.cs

```diff
@@ -7,18 +7,60 @@
 using System.Collections.ObjectModel;
```

Comment:
This can be re-used for other C# projects too. I suggest it be in the common folder. It would be nice if we could have a C# common project to manage PowerToys JSON settings so they can also be in the TweakUI settings View. 

![image](https://user-images.githubusercontent.com/58791731/74311839-3239d100-4d25-11ea-9dea-af91950d4c3e.png)

Reply:
An example of this is the cpp [settings_object.h/cpp](https://github.com/arjunbalgovind/PowerToys/blob/user/arbalgov/JSONsettings/src/common/settings_objects.h) and [settings_helper.h/cpp](https://github.com/arjunbalgovind/PowerToys/blob/user/arbalgov/JSONsettings/src/common/settings_helpers.h)

Reply:
The plan is to eventually shift the Settings out of the C# code and into the C++ dllmain project and in the TweakUI settings view. Since shifting to the TweakUI and C++ would take more dev time it might not be ready for the 0.16 release, which is why @crutkas suggested leaving the settings within the ImageResizer application but changing the settings from XML to JSON so that there would be no breaking change in the future. We would also add some verbiage in the view you added above to mention that the ImageResizer settings are within the applicaton. To minimize dev effort it would make most sense to migrate the ImageResizer settings directly to WinUI when that change is made for the TweakUI settings window.
That's why I didn't make this common code as its a one-off exception which needs to be migrated to C++ in the future.

Reply:
Okay, you can still leave it inside image resizer but use an external common library to manage JSON settings.

Reply:
I don't know if we should make the C# JSON code as a common library since:
1. Only Save() and Reload() functions are added for the JSON read/write but some of the code in it is specific to the properties present in the ImageResizer Settings.
2. Since the migration to WinUI settings will involve shifting these ImageResizer settings to C++ to be visible within the PowerToys settings menu this code will no longer be used and we want to avoid using settings in other C# projects as well and keep it all within the C++ code base.

@ryanbodrug-microsoft any suggestions?